src/BrowserDetector.NetCore/Detection/IBrowserDetector.cs

```diff
@@ -8,6 +8,6 @@ public interface IBrowserDetector
         /// <summary>
         /// Gets the browser for the current HTTP request.
         /// </summary>
-        IBrowser? Browser { get; }
+        IBrowser Browser { get; }
```

Comment:
This is a breaking change since this type is public.

Reply:
i reversed this and added #nullable enable to the files

Reply:
Its also possible to enable this at the project-file. then, is necessary to refactore the browser classes like firefox, chrome and so on